inputfiles/overridingTypes.jsonc

```diff
@@ -3426,4 +3433,4 @@
             }
         }
     ]
-}
+}
```

Comment:
EOF is accidentally removed 👀

Reply:
No idea how that happened... should be fixed now.